lib/lago_utils/lago_utils/version.rb

```diff
@@ -7,12 +7,11 @@ class Version
     VERSION_FILE = Rails.root.join("LAGO_VERSION")
     GITHUB_BASE_URL = "https://github.com/getlago/lago-api"
 
+    Result = Data.define(:number, :github_url)
+
     class << self
       def call(default:)
-        OpenStruct.new(
```

Comment:
You can remove the `require "ostruct"` at the top of the file. (I cna't comment unmodified lines apparently